packages/mux-player/package.json

```diff
@@ -9,12 +9,29 @@
   "jsdelivr": "./dist/mux-player.js",
   "exports": {
     ".": {
+      "types": "./dist/types/index.d.ts",
       "import": "./dist/index.mjs",
       "require": "./dist/index.cjs.js",
       "default": "./dist/index.cjs.js"
     },
-    "./themes/microvideo": "./themes/microvideo.js",
-    "./themes/minimal": "./themes/minimal.js"
+    "./themes/microvideo": {
+      "import": "./dist/themes/microvideo/index.mjs",
+      "require": "./dist/themes/microvideo/index.cjs.js",
+      "browser": "./dist/themes/microvideo/index.mjs",
+      "default": "./dist/themes/microvideo/index.cjs.js"
+    },
+    "./themes/minimal": {
+      "import": "./dist/themes/minimal/index.mjs",
+      "require": "./dist/themes/minimal/index.cjs.js",
+      "browser": "./dist/themes/minimal/index.mjs",
+      "default": "./dist/themes/minimal/index.cjs.js"
+    },
+    "./themes/classic": {
+      "import": "./dist/themes/classic/index.mjs",
+      "require": "./dist/themes/classic/index.cjs.js",
+      "browser": "./dist/themes/classic/index.mjs",
+      "default": "./dist/themes/classic/index.cjs.js"
+    }
```

Comment:
this looks good!

I hope we can deprecate `.mjs` soon. it's really a stone in my garden

we figured out a way to only use `.js` in Media Chrome